regression/pages/whitelabel/const.py

```diff
@@ -0,0 +1,327 @@
+# -*- coding: utf-8 -*-
+"""
+List of constants to be used throughout the tests
+"""
+import os
+import uuid
+
+# Get HTTP Authentication credentials from environment variables
+AUTH_USERNAME = os.environ['BASIC_AUTH_USER']
+AUTH_PASSWORD = os.environ['BASIC_AUTH_PASSWORD']
+
+# Get API Access token
+
+ACCESS_TOKEN = os.environ['ACCESS_TOKEN']
+
+# Select the Org for which to run the tests, Default is MITProfessionalX
+ORG = os.getenv('ORG', 'MITProfessionalX')
+
+# GMAIL account
+GMAIL_USER = os.environ['GMAIL_USER']
+DEFAULT_GMAIL_USER = GMAIL_USER.format("")
+GMAIL_PASSWORD = os.environ['GMAIL_PASSWORD']
+
+# Global password
+PASSWORD = os.environ['GLOBAL_PASSWORD']
+
+BASIC_AUTH = AUTH_USERNAME + ":" + AUTH_PASSWORD + "@"
+
+# Organization Based Settings
+##############################################################################
+# E-commerce raw url
+RAW_ECOMMERCE_URL = {
+    'HarvardXPLUS': u'https://{}stage-payments.harvardxplus.harvard.edu/',
+    'MITProfessionalX': u'https://{}payments.mitprofessionalx-stage.mit.edu/'
+}
+
+# BASIC raw URL
+RAW_URL = {
+    'HarvardXPLUS': u'https://{}stage-courses.harvardxplus.harvard.edu/',
+    'MITProfessionalX': u'https://{}mitprofessionalx-stage.mit.edu/'
+}
+
+# E-commerce urls
+ECOMMERCE_URL_WITHOUT_AUTH = RAW_ECOMMERCE_URL[ORG].format("")
+
+ECOMMERCE_URL_WITH_AUTH = RAW_ECOMMERCE_URL[ORG].format(BASIC_AUTH)
+
+# BASIC URLs
+URL_WITHOUT_AUTH = RAW_URL[ORG].format("")
+
+URL_WITH_AUTH = RAW_URL[ORG].format(BASIC_AUTH)
+
+ECOMMERCE_API_URL = ECOMMERCE_URL_WITHOUT_AUTH + 'api/v2/'
+
+CYBERSOURCE_CHECKOUT_URL = \
+    u'https://testsecureacceptance.cybersource.com/checkout'
+
+PROF_COURSE_ID = {
```

Comment:
If we need to test actual courses, that is ok. I just think we would remove them from the code and pass it in as a variable (for example).